src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java

```diff
@@ -36,13 +37,30 @@ public void onEnable() {
         if (!new File(getDataFolder(), "config.yml").exists())
             saveDefaultConfig();
 
+        if (!getConfig().contains("options.need-wrench-to-break-machines")) {
+            getConfig().set("options.need-wrench-to-break-machines", false);
+            saveConfig();
+        }
+
+        if (!getConfig().contains("options.wrench-failure-chance")) {
+            getConfig().set("options.wrench-failure-chance", 0.0);
+            saveConfig();
+        }
+
         final Metrics metrics = new Metrics(this, 7111);
         setupCustomMetrics(metrics);
 
         if (getConfig().getBoolean("options.auto-update") && getDescription().getVersion().startsWith("DEV - ")) {
```

Comment:
new line

Reply:
? the auto update?

Reply:
&& getDescription().getVersion().startsWith("DEV - ")) {